core/client/components/gh-trim-focus-input.js

```diff
@@ -5,16 +5,20 @@ var TrimFocusInput = Ember.TextField.extend({
     attributeBindings: ['autofocus'],
 
     autofocus: Ember.computed(function () {
```

Comment:
I don't think the focus property should change, and the attribute binding will "get" autofocus during create so I wouldn't expect there to be an issue in any case. 

As far as convention goes, I think a CP that doesn't observe a property is legitimate if you really don't want its value to auto update.